factories/tags.rb

```diff
@@ -68,6 +68,10 @@
     sequence(:name) { |n| "Unsorted Tag #{n}" }
   end
 
+  factory :archive_warning do
```

Comment:
We don't add a factory for warnings because there is a set number of them and you can refer to them by names, e.g. `ArchiveConfig.WARNING_DEFAULT_TAG_NAME`.

Reply:
Sounds good! I wasn't sure since there weren't other tests using archive warnings. But it makes sense, since the tag will be auto-created as part of work creation if it doesn't already exist. I'll go ahead and remove this!

Reply:
Resolved in da1fb7273